.github/workflows/make_release.yaml

```diff
@@ -0,0 +1,91 @@
+name: Make a Beets Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version of the new release'
+        required: true
+
+jobs:
+  increment_version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Install pandoc
+      run: sudo apt update && sudo apt install pandoc -y
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.9"
```

Comment:
```suggestion
        python-version: "3.11"
```

Reply:
Python 3.9 is still our minimally supported version for the moment.